python/python/bystro/ancestry/train.py

```diff
@@ -280,8 +406,8 @@ def _filter_variants_for_fst(genotypes: pd.DataFrame, labels: pd.DataFrame) -> p
 def _load_dataset() -> tuple[pd.DataFrame, pd.DataFrame]:
     """Return the final dataset consisting of genotype matrix, labels."""
     genotypes = _load_genotypes()
-    labels = _load_label_data(genotypes.index)
-    genotypes, labels = _filter_samples_for_relatedness(genotypes, labels)
+    labels = load_label_data(genotypes.index)
+    genotypes, labels = filter_samples_for_relatedness(genotypes, labels)
     genotypes = _filter_variants_for_maf(genotypes)
```

Comment:
MAF filtering is already a filter in the preprocess step - is there a reason we want to repeat that preprocess step here?

Reply:
I think it may be easier to discuss offline?